docs/kb/relayer.md

```diff
@@ -23,6 +23,21 @@ The optional `--advanced` flag lets you configure port and version for the custo
 
 By default, relayer configuration is stored in `$HOME/.relayer/`.
 
+## Remove existing relayers
+
+If you previously used the Starport relayer, follow these steps to remove existing relayer and Starport configurations:
+
+1. Stop your blockchain or blockchains.
+2. Delete previous configuration files:
+
+    ```bash
+    rm -rf ~/.starport/relayer
+    ```
+
+    If existing configurations do not exist, the command returns `no matches found` and no action is taken.
```

Comment:
I think if you use the `-f` flag the return will be always empty in most shells. 

```suggestion
2. If a previous configuration exist, you should delete the config files:

    ```bash
    rm -rf ~/.starport/relayer
    ```
```